src/core/inspect/kernel.rs

```diff
@@ -165,15 +165,21 @@ impl KernelInspector {
                 return Ok(Some(parse_kconfig(&file)?));
             }
 
-            // If the above didn't work, try reading from /boot/config-$(uname -r).
-            let config_file = format!("/boot/config-{}", release);
-            if let Ok(file) = fs::read_to_string(&config_file) {
-                return Ok(Some(parse_kconfig(&file)?));
+            // If the above didn't work, try reading from known paths.
+            let paths = vec![
+                format!("/boot/config-{}", release),
+                // CoreOS & friends.
+                format!("/lib/modules/{}/config", release),
+                // Toolbox on CoreOS & friends.
+                format!("/run/host/usr/lib/modules/{}/config", release),
```

Comment:
it's ok keeping this, but we might theoretically (in practice this should not be true) end up chasing different non-standard paths.
Maybe at some point a `--kconfig` might make sense with "/boot/config-{}" as default attempt if not specified?